data/compatible_products.yml

```diff
@@ -4,95 +4,109 @@
 #    items:
 #    - name: Eclipse Glassfish
 #      vendor: Eclipse Foundation
-#      version: 5.1.0
 #      image: '/images/glassfish.png'
 #      download: 'https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip'
+#      versions:
+#      - version: 5.1.0
+#        compatibility: 'https://eclipse-ee4j.github.io/glassfish/certifications/jakarta-full-profile/8.0/TCK-Results'
 
 sets:
-  - title: Jakarta EE 8 Full Platform Compatible Products
+  - title: Jakarta EE 8 Platform Compatible Products
     items:
     - name: Payara Server 
       vendor: Payara Services Limited
-      version: 5.193.1
       image: '/images/compatible_products/payara_server.png'
       download: 'https://www.payara.fish/software/downloads/'
-      compatibility: 'https://github.com/eclipse-ee4j/jakartaee-platform/issues/119'
+      versions:
+      - version: 5.193.1
+        compatibility: 'https://docs.payara.fish/community/docs/5.2020.2/jakartaee-certification/5.193/tck-results-full-5.193.1.html'
 
     - name: Eclipse Glassfish
       vendor: Eclipse Foundation
-      version: 5.1.0, Full Profile
       image: '/images/compatible_products/glassfish_logo.svg'
       download: 'https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip'
-      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/440'
+      versions:
+      - version: 5.1.0
+        compatibility: 'https://eclipse-ee4j.github.io/glassfish/certifications/jakarta-full-profile/8.0/TCK-Results'
 
     - name: Open Liberty
       vendor: IBM Corporation
-      version: 19.0.0.6
       image: '/images/compatible_products/ibm_open_liberty.png'
       download: 'https://openliberty.io/downloads/'
-      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/439'
+      versions:
+      - version: 19.0.0.6
+        compatibility: 'https://openliberty.io/certifications/jakartaee/platform/8/TCKResults.html'
 
     - name: WildFly
       vendor: Red Hat
-      version: 18.0.0.Final
       image: '/images/compatible_products/red-hat-wildfly-logo.png'
       download: 'https://wildfly.org/downloads'
-      compatibility: 'https://github.com/jakartaee/jakarta.ee/issues/508'
+      versions:
+      - version: 18.0.0
+        compatibility: 'https://github.com/wildfly/certifications/blob/EE8/WildFly_18.0.0.Final/jakarta-web-profile.adoc'
```

Comment:
This link doesn't look right.  The Full Platform certification is pointing at Web Profile results?  I'd check with RH before merging.

Reply:
Egad, good catch. That should be https://github.com/wildfly/certifications/blob/EE8/WildFly_18.0.0.Final/jakarta-full-platform.adoc